src/Services/Location/Locations.API/Infrastructure/Middlewares/ByPassAuthMiddleware.cs

```diff
@@ -62,8 +62,7 @@ public async Task Invoke(HttpContext context)
                     new Claim("emails", currentUserId),
                     new Claim("name", "Test user"),
                     new Claim("nonce", Guid.NewGuid().ToString()),
-                    new Claim("ttp://schemas.microsoft.com/identity/claims/identityprovider", "ByPassAuthMiddleware"),
-                    new Claim("nonce", Guid.NewGuid().ToString()),
+                    new Claim("http://schemas.microsoft.com/identity/claims/identityprovider", "ByPassAuthMiddleware"),
                     new Claim("sub", "1234"),
```

Comment:
Hey @anjoy8  - wouldn't be better to update the "sub" claim to `new Claim("sub", currentUserId)` for other `ByPassAuthMiddleware.cs` classes as well. Because I could see that `_identityService.GetUserIdentity()` is getting used for other scenarios as well ? For e.g `CreateOrUpdateUserLocationAsync` method in `LocationsController` class.